README.md

```diff
@@ -8,26 +8,27 @@
 
 **ONNX Runtime** is a performance-focused inference engine for ONNX (Open Neural Network Exchange) models.
 
-Models in the Tensorflow, Keras, PyTorch, scikit-learn, CoreML, and other popular [supported formats](https://github.com/onnx/tutorials#converting-to-onnx-format) can be converted to the standard [ONNX](https://onnx.ai) format, providing framework interoperability and helping to maximize the reach of hardware optimization investments. This provides a solution for systems to integrate a single inference engine to support models trained from a variety of frameworks, while taking advantage of specific hardware accelerators where available.
+Models trained with Tensorflow, Keras, PyTorch, scikit-learn, CoreML, and other popular [supported training frameworks](https://github.com/onnx/tutorials#converting-to-onnx-format) can be converted to the standard [ONNX](https://onnx.ai) format, providing framework interoperability and helping to maximize the reach of hardware optimization investments. This provides a solution for systems to integrate a single inference engine to support models trained from a variety of frameworks, while taking advantage of specific hardware accelerators where available.
 
 ONNX Runtime was designed with a focus on performance and scalability in order to support heavy workloads in high-scale production scenarios. It also has extensibility options for compatibility with emerging hardware developments.
 
-ONNX Runtime stays up to date with the ONNX standard and supports all operators from the ONNX v1.2+ spec and is backwards compatible with older versions. Please refer to [this page](docs/Versioning.md) for ONNX opset compatibility details.
+ONNX Runtime stays up to date with the ONNX standard and supports all operators from the ONNX v1.2+ spec with both forwards and backwards compatibility. Please refer to [this page](docs/Versioning.md) for ONNX opset compatibility details.
 
 
 
 ***
 # Table of Contents
-* **[Functional Overview](#functional-overview)**
+* **[Functional Overview](#overview)**
   * [Key Features](#key-features)
   * [Performance Focused Design](#performance-focused-design)
     * [Performance Tuning](./docs/ONNX_Runtime_Perf_Tuning.md)
   * [Extensibility Options](#extensibility-options)
 * **[Installation](#installation)**
-  * [API Documentation](#api-documentation)
+  * [API Documentation](#apis-documentation)
```

Comment:
broken link.
should be #api-documentation.

is there some kind of auto-correct happening? noticed lot of spurious changes.

Reply:
I mistakenly had an older version of the readme in my fork so it pushed older changes that were fixed. Resync'd it with the master now.